src/modules/IscsiLioData.rb

```diff
@@ -599,7 +599,8 @@ def GetIpAddr
 
       ip_list.reject! do |address|
         address.start_with?("127.") ||  # local IPv4
-          address.start_with?("::1") # local IPv6
+          address.start_with?("::1") || # local IPv6
+          address.start_with?("fe80:")  # scope link IPv6
       end
 
       ip_list = [""] if ip_list.empty?
```

Comment:
ehm ... very 'interesting' ... and undocumented. In general I'd expected empty array when no ip was found.